Makefile

```diff
@@ -62,7 +60,7 @@ endif
 
 OPTFLAGS = -O2 -std=c++11 -fno-rtti
 
-CCFLAGS = -Wall $(OPTFLAGS) $(OPT_ENABLE_DEBUGGER_SUPPORT)
+CCFLAGS = -Wall $(CXXFLAGS) $(OPTFLAGS) $(OPT_ENABLE_DEBUGGER_SUPPORT)
```

Comment:
unfortunately, this will break builds for most other platforms.

Reply:
Thanks for having a look.  You probably meant the -lv8_libplatform removal, which is indeed needed (also on Fedora, I was just mistaken by Fedora bug https://bugzilla.redhat.com/show_bug.cgi?id=1517657).

Reply:
yeah, I meant to comment on the `v8_libplatform` line.